### PR TITLE
Release

### PR DESCRIPTION
**Pull request by adborden**

### CODE REVIEW

File: application/helpers/api_helper.php [hunk 1]
```diff
@@ -305,6 +305,56 @@ function filter_json( $source_datajson, $dataset_array = false ) {
...
+        }
+
+
+    }
+    // filter xss
+    if ($url && function_exists('xss_clean')) {
+        $url = xss_clean($url);
+    }
+    return $url;
+}
```
Comment (afeld) [inline application/helpers/api_helper.php]: Any change to the function?

File: application/controllers/campaign.php [hunk 2]
```diff
@@ -917,7 +917,7 @@ public function status($id = null, $component = null, $selected_milestone = null
                         $expected_datajson_url = urldecode($url_override);
                     }
 
-                    $expected_datajson_url = $this->campaign->filter_remote_url($expected_datajson_url);
+                    $expected_datajson_url = filter_remote_url($expected_datajson_url);
```
Comment (afeld) [inline application/controllers/campaign.php]: Helpers get included everywhere? No namespace or an anything like that?

File: application/helpers/api_helper.php (hunk 1 above)
Comment (adborden) [reply application/helpers/api_helper.php]: Nope, no change was needed.

File: application/controllers/campaign.php (hunk 2 above)
Comment (adborden) [reply application/controllers/campaign.php]: I'm pretty sure this is what does it, but honestly not sure.

https://github.com/GSA/project-open-data-dashboard/blob/f0c63cc40ec8233428bdaa9fbf790b57011b5897/application/controllers/campaign.php#L10

Comment (adborden) [reply application/controllers/campaign.php]: These docs seem to back this up https://www.codeigniter.com/userguide3/general/helpers.html

**Pull request merged**